src/fu-util-common.c

```diff
@@ -17,15 +17,24 @@
 #define SYSTEMD_OBJECT_PATH		"/org/freedesktop/systemd1"
 #define SYSTEMD_MANAGER_INTERFACE	"org.freedesktop.systemd1.Manager"
 #define SYSTEMD_FWUPD_UNIT		"fwupd.service"
+#define SYSTEMD_SNAP_FWUPD_UNIT		"snap.fwupd.fwupd.service"
```

Comment:
Out of interest, if the "upstream" service name was `org.freedesktop.fwupd.service` what would the snap name be?